python/postprocessing/modules/btv/btagSFProducer.py

```diff
@@ -130,6 +130,15 @@ def __init__(
                         2: "incl"   # light
                     },
                     'supported_wp': ["L", "M", "T", "shape_corr"]
+                },
```

Comment:
If possible you could add the DeepCSV UL SF around this line:
https://github.com/cms-nanoAOD/nanoAOD-tools/blob/ff0b5d7ceccc98b5f1317ffaa07e535813a2ccae/python/postprocessing/modules/btv/btagSFProducer.py#L104

Reply:
Hi @gouskos, I added the DeepCSV and DeepJet b-tag SF for UL2017 and the DeepCSV b-tag SF for UL2018 and changed btagSFProducer.py accordingly.